site/content/docs/02-template-syntax.md

```diff
@@ -627,7 +627,7 @@ On `<input>` elements with `type="file"`, you can use `bind:files` to get the [`
 
 ---
 
-`bind:` can be used together with `on:` directives. The order that they are defined in determines the value of the bound variable when the event handler is called.
+If you're using `bind:` directives together with `on:` directives, their order can affect what value the bound variable has when the event handler is called.
```

Comment:
This is a bit awkwardly worded in my opinion. I'd make it closer to the original:
```suggestion
If you're using `bind:` directives together with `on:` directives, the order that they're defined in affects the value of the bound variable when the event handler is called.
```